src/ReactEntity.js

```diff
@@ -36,7 +43,7 @@ class ReactEntity {
   }
 
   applyEntityConstructor(field, data) {
-     if (!data) return;
+     if (data === undefined || data === null) return;
```

Comment:
Before that we had this;

    if (!data) return;

so, whenever we set a value to `false` it was returning a `undefined`. 
`(data === false)`

so, when you set a value with `undefined` or `null`, I just change to return the same `data`